packages/react-components/react-tree/src/components/TreeItem/useTreeItem.tsx

```diff
@@ -241,8 +241,8 @@ export function useTreeItem_unstable(props: TreeItemProps, ref: React.Ref<HTMLDi
         'aria-level': level,
         'aria-checked': selectionMode === 'multiselect' ? checked : undefined,
         // Casting: when selectionMode is 'single', checked is a boolean
-        'aria-selected': selectionMode === 'single' ? !!checked : undefined,
-        'aria-expanded': ariaSelected !== undefined ? ariaSelected : itemType === 'branch' ? open : undefined,
+        'aria-selected': ariaSelected !== undefined ? ariaSelected : selectionMode === 'single' ? !!checked : undefined,
+        'aria-expanded': itemType === 'branch' ? open : undefined,
```

Comment:
`aria-expanded` from user prop will be overridden.
```suggestion
        'aria-expanded': props['aria-expanded'] ?? (itemType === 'branch' ? open : undefined),
```

Reply:
added in https://github.com/microsoft/fluentui/pull/31117/commits/8b4337c108d7367763b3356fb15b3bbed0ddd571